spec/rails_helper.rb

```diff
@@ -8,60 +8,28 @@
 require 'rspec/rails'
 require 'capybara/rspec'
 require 'capybara/rails'
-# Add additional requires below this line. Rails is not loaded until this point!
```

Comment:
Can we keep this inline with the generator output as per https://github.com/alphagov/datagovuk_find/pull/608?

Reply:
Done - I was actually going to do another PR to replicate what you did in Find. But I don't think we have time now.